src/services/mirage.js

```diff
@@ -1,28 +1,51 @@
-import { Server } from 'miragejs';
-import { mockGroup, mockGroupList } from '../pages/groups/models/mockData';
-import KOptions from '../komunitin.json';
+import { Server } from "miragejs";
+import {
+  mockGroup,
+  mockGroupList,
+  mockCategoryList,
+  mockCurrency,
+  mockOfferList
+} from "../pages/groups/models/mockContent/mockData";
+import KOptions from "../komunitin.json";
 
-console.debug('Mirage activated');
+console.debug("Mirage activated");
 
-new Server({
+const urlSocial = KOptions.apis.social;
+const urlAccounting = KOptions.apis.accounting;
 
+new Server({
   // Take the Base url from mockData.ts
-  urlPrefix: KOptions.apis.social,
+  // urlPrefix: KOptions.apis.social,
```

Comment:
Prefiero entonces borrar la línea, que sino va quedando el código lleno de líneas comentadas sin mucho sentido.  Por otro lado, dentro de routes() creo que puedes ir cambiando la variable `this.namespace` o incluso `this.urlPrefix` entre this.get y this.get. Quiero decir:
```
routes() {
  this.namespace = KOptions.apis.social
  this.get(...)
  this.get(...)
  this.namespace = KOptions.apis.accouting
  this.get(...)
}
```
Pero bueno, como lo has hecho también está igual de bien, es lo mismo!

Reply:
Esta parte quiero revisarla en su momento.